### DISCUSSION TITLE
Can't import styles in Angular component

### DISCUSSION THREAD

**Discussion by stasonn777** [nolimits4web/swiper#5767] [Q&A]
Hello! I'm trying to import swiper styles to component in my Angular app but its doesn't work.
Whe I add @import '~swiper/swiper.scss'; to main scss file its working, but not in component styles file by import 'swiper/scss';. Also it's doesn't work when importing to component.ts file. 
Is it possible to import styles to component? 

"dependencies": {
    "@angular/animations": "~13.3.0",
    "@angular/common": "~13.3.0",
    "@angular/compiler": "~13.3.0",
    "@angular/core": "~13.3.0",
    "@angular/forms": "~13.3.0",
    "@angular/platform-browser": "~13.3.0",
    "@angular/platform-browser-dynamic": "~13.3.0",
    "@angular/router": "~13.3.0",
    "rxjs": "~7.5.0",
    "swiper": "^8.2.2",
    "tslib": "^2.3.0",
    "zone.js": "~0.11.4"
  },
  "devDependencies": {
    "@angular-devkit/build-angular": "~13.3.7",
    "@angular/cli": "~13.3.7",
    "@angular/compiler-cli": "~13.3.0",
    "@types/jasmine": "~3.10.0",
    "@types/node": "^12.11.1",
    "jasmine-core": "~4.0.0",
    "karma": "~6.3.0",
    "karma-chrome-launcher": "~3.1.0",
    "karma-coverage": "~2.1.0",
    "karma-jasmine": "~4.0.0",
    "karma-jasmine-html-reporter": "~1.7.0",
    "typescript": "~4.6.2"
  }

**Comment by vltansky** [comment]
try `@import "~swiper/scss"` in `some.component.scss` file

**Reply by Eremeow138** [reply]
same problem(
